src/quicklookUI.cs

```diff
@@ -168,5 +168,9 @@ interface QLPreviewingController {
 #endif
 		[Export ("preparePreviewOfSearchableItemWithIdentifier:queryString:completionHandler:")]
 		void PreparePreviewOfSearchableItem (string identifier, string queryString, Action<NSError> ItemLoadingHandler);
+
+		[Mac (10,15, onlyOn64: true)]
```

Comment:
In preparation of https://github.com/xamarin/xamarin-macios/pull/6595 please remove `, onlyOn64: true`